_topic_maps/_topic_map.yml

```diff
@@ -1525,22 +1525,38 @@ Topics:
   Topics:
   - Name: Understanding multiple networks
     File: understanding-multiple-networks
-  - Name: Understanding user defined networks
-    File: understanding-user-defined-network
-  - Name: Configuring an additional network
-    File: configuring-additional-network
+  - Name: Primary networks
+    Dir: primary_networks
+    Distros: openshift-enterprise, openshift-origin
+    Topics:
+    - Name: UserDefinedNetwork CR
+      File: about-user-defined-networks
+    - Name: NetworkAttachmentDefinition CR
+      File: about-primary-nwt-nad
+  - Name: Secondary networks
+    Dir: secondary_networks
+    Distros: openshift-enterprise, openshift-origin
+    Topics:
+    - Name: Creating secondary networks on OVN-Kubernetes
+      File: creating-secondary-nwt-ovnk
+    - Name: Creating secondary networks with other CNI
```

Comment:
I think technically it'd be "with other CNI plugins" 

> Creating secondary networks with other CNI plugins

Otherwise, it's kind of an incomplete though from a CNI viewpoint